install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.8-kube-122-api-removals-in-4.9: |
+      Kubernetes 1.22 and thus OpenShift 4.9 remove several APIs which require admin consideration. Please see
+      https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-deprecated-removed-features.
```

Comment:
Hmm.  Do we have a UX for exposing these outside of the console and `oc` `Upgradeable` handling?  I'll check how the console handles linking, to see if we need a space between the URI and the trailing `.` or not...

Reply:
Turns out the lack of space between the URI and trailing `.` is fine; see the screenshot [here][1] (which is getting that condition linkified).

[1]: https://github.com/openshift/console/pull/9955#issue-984101602